zap/src/scan.py

```diff
@@ -374,7 +375,96 @@ def clean_uri_path(xml_report):
         uri.text = urlunparse(r)
     tree.write(xml_report)
 
-def upload_googledrive(scan_type, zap_filename, codedx_project, report_file, slack_token, slack_channel):
+def get_codedx_findings_json(cdx,codedx_project):
+    """
+    Raw report findings_json
+    """
+    project_id = cdx.get_project_id(codedx_project)
+    options = ["descriptor",
+               "issue",
+               "descriptions",
+               "results.active",
+               "results.descriptor",
+               "results.descriptions",
+               "results.metadata",
+               "results.variants.with-body",
+               "last-comment-action",
+               "metadata,"
+               "tags",
+               "aggregations.tool-summary",
+               "triage-time"]
+    config = {
+                "filter": {
+                    "~status": [
+                    7
+                    ],
+                    "globalConfig":{"ignoreArchived":True}
+                },
+                "pagination":{"perPage":500,"page":1}
+                }
+    return cdx.get_finding_table(project_id, options, config,  )
+
+def hail_compliance_export(results_json, project_name):
+    """
+    Helper function to create a CSV file in the POAM format of the current raw findings
+    """
+    csv_headers = ["status",
+                   "ids",
+                   "Weakness Name",
+                   "Weakness Description",
+                   "Weakness Detector Source",
+                   "Weakness Source Identifier",
+                   "Asset Identifier",
+                   "Original Detection Date",
+                   "Scheduled Completion Date",
+                   "Original Risk Rating"]
+    # create the csv line based on the columns above.
+    results = {}
+    for result in results_json:
+        result_line = []
+        name = result.get("descriptor").get("name")
+        status = result.get("statusName")
+        result_line.append(status)
+        result_line.append("")
+        result_line.append(name)
+        result_line.append(result.get("descriptions").get("general").get("content"))
+        result_line.append("put file location here")
```

Comment:
Is this a placeholder for a person to fill in?

Reply:
Good catch. It can be a place holder, but I think I can just stick the raw report pdf name in there.

Reply:
Fixed this, so the raw report filename is now in the csv.